src/plugins/intel_cpu/src/memory_desc/cpu_memory_desc_utils.h

```diff
@@ -42,11 +43,11 @@ class MemoryDescUtils {
     static DnnlBlockedMemoryDesc convertToDnnlBlockedMemoryDesc(const MemoryDesc& desc);
 
     /**
-     * @brief Converts InferenceEngine::TensorDesc to CpuBlockedMemoryDesc
-     * @param desc InferenceEngine::TensorDesc to be converted
-     * @return converted CpuBlockedMemoryDesc
+     * @brief Converts MemoryDesc to BlockedMemoryDesc
+     * @param desc MemoryDesc to be converted
+     * @return converted BlockedMemoryDesc
      */
-    static CpuBlockedMemoryDesc convertToCpuBlockedMemoryDesc(const InferenceEngine::TensorDesc& desc);
+    static std::shared_ptr<BlockedMemoryDesc> convertToBlockedMemoryDesc(const std::shared_ptr<MemoryDesc> &desc);
```

Comment:
Is this method in use anywhere? If not, please remove it.

Reply:
It is used by:
https://github.com/openvinotoolkit/openvino/blob/44f7bf7e3ff685b12bc58b36ea37941001e3bf43/src/plugins/intel_cpu/src/cpu_memory.cpp#L31-L34